app/routes/application.js

```diff
@@ -143,26 +142,6 @@ export default Route.extend(
 			}
 		},
 
-		redirect(model, transition) {
```

Comment:
I guess this is still in progress. Please don't forget about reimplementing.

Reply:
do you remember what exactly this was handling?

Reply:
http://the-teen-wolf.wikia.com/wiki/Scott_McCall redirect to http://teen-wolf-pack.wikia.com/wiki/Scott_McCall (primary domain)

Reply:
Related: https://wikia-inc.atlassian.net/browse/XW-3198

Reply:
ok, reapplied